shell/platform/fuchsia/flutter/fuchsia_intl.cc

```diff
@@ -0,0 +1,126 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
Please use the usual Flutter license block like in the files next to this TU.

Reply:
Will do. I will just assume that your suggested approach is fine.